public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -377,144 +375,110 @@ include ../_util-fns
-- var lang = current.path[1]
-- var a_decorator = lang == 'dart' ? 'an annotation' : 'a decorator'
-- var decorated = lang == 'dart' ? 'annotated' : 'decorated'
-- var any_decorator = lang == 'dart' ? '' : 'TypeScript generates metadata for any class with a decorator, and any decorator will do.'
-.callout.is-helpful
-  header Suggestion: add @Injectable() to every service class
-  :marked
-    We recommend adding `@Injectable()` to every service class, even those that don't have dependencies
-    and, therefore, do not technically require it. Here's why:
-  ul(style="font-size:inherit")
-    li <b>Future proofing:</b> No need to remember <code>@Injectable()</code> when we add a dependency later.
-    li <b>Consistency:</b> All services follow the same rules, and we don't have to wonder why #{a_decorator} is missing.
-
-  :marked
-    Although we recommend applying `@Injectable` to all service classes, do not feel bound by it.
-    Some developers prefer to add it only where needed and that's a reasonable policy too.
 
-.l-sub-section
-  :marked
-    The `HeroesComponent` has an injected dependency too. Why don't we add `@Injectable()` to the `HeroesComponent`?
+//- FIXME refer to Dart API when that page becomes available.
+- var injMetaUrl = 'https://angular.io/docs/ts/latest/api/core/index/InjectableMetadata-class.html';
+h3#injectable Why @Injectable()?
+:marked
+  **<a href="#{injMetaUrl}">@Injectable()</a>** marks a class as available to an
+  injector for instantiation. Generally speaking, an injector will report an
+  error when trying to instantiate a class that is not marked as
+  `@Injectable()`.
+
+block injectable-not-always-needed-in-ts
+  .l-sub-section
+    :marked
+      As it happens, we could have omitted `@Injectable()` from our first
+      version of `HeroService` because it had no injected parameters.
+      But we must have it now that our service has an injected dependency.
+      We need it because Angular requires constructor parameter metadata 
+      in order to inject a `Logger`.
+
+  .callout.is-helpful
+    header Suggestion: add @Injectable() to every service class
+    :marked
+      We recommend adding `@Injectable()` to every service class, even those that don't have dependencies
+      and, therefore, do not technically require it. Here's why:
+
+    ul(style="font-size:inherit")
+      li <b>Future proofing:</b> No need to remember <code>@Injectable()</code> when we add a dependency later.
+      li <b>Consistency:</b> All services follow the same rules, and we don't have to wonder why #{_a} #{_decorator} is missing.
+
+:marked
+  Injectors are also responsible for instantiating components
+  like `HeroesComponent`. Why haven't we marked `HeroesComponent` as
+  `@Injectable()`?
+
+  We *can* add it if we really want to. It isn't necessary because the
+  `HeroesComponent` is already marked with `@Component`, and this
+  !{_decorator} class (like `@Directive` and `@Pipe` which we will study later
```

Comment:
"study" sounds so serious. Also, "which" (when introducing a phrase/clause) should have a comma before it; if a comma doesn't seem right, then you should probably be using "that" instead.

```
` which we will study later on
```
->
```
`, which we'll see later
```